src/bindings/lua/wreck.lua

```diff
@@ -646,7 +646,7 @@ end
 -- Return a list and hash of active jobs by kvs path
 local function joblist_active (arg)
     local f = arg.flux
-    local r, err = f:rpc ("job.list", {})
+    local r, err = f:rpc ("job.list", {max = arg.max})
```

Comment:
Sorry, was working on adding this and the include/exclude list to the rpc but didn't have a change to push changes today.

Note that if you pass `max` here it probably doesn't need to be checked again below.

I'll get to the include, exclude lists tomorrow unless you are already working on it?

Reply:
Would be happy to hand that off (and drop my last commit here in favor of yours).  Thanks!